src/Platforms.jl

```diff
@@ -158,7 +158,10 @@ const ARCHITECTURE_FLAGS = Dict(
             #"power9"  => ["-mcpu=power9", "-mtune=power9"],
             # Eventually, we'll support power10, once we have compilers that support it.
             #"power10" => ["-mcpu=power10", "-mtune=power10"],
-        )
+        ),
+        "riscv64" => OrderedDict(
+            "riscv64" => [],
```

Comment:
We probably need to come up with a basenline ISA/ABI to target here.  `-march=rv64g -mabi=lp64d`?

Reply:
@maleadt used `rv64gc_zba_zbb ` in https://github.com/JuliaLang/julia/pull/56105.

Reply:
That was my local build for my VisionFive2. For the more generic builds at https://github.com/maleadt/julia/releases, I used `rv64gc` (aka. `generic-rv64,i,m,a,f,d,zicsr,zifencei,c` for `JULIA_CPU_TARGET`). I haven't put a lot of thought in it, but `rv64gc` seems like a good baseline, with support for other profiles (e.g. `rva22`) through platform augmentation tags.

Reply:
Using your suggestion.

Reply:
According to https://wiki.gentoo.org/wiki/RISC-V_Multilib `-march=rv64gc -mabi=lp64d` is the default ISA/ABI used by GCC, so it sounds like a good default baseline. I also found https://www.sifive.com/blog/all-aboard-part-1-compiler-args to be an interesting read about compiler flags for RISC-V (all those extensions are a mess)